docs/pages/build-reference/tests.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Running E2E tests on EAS Build
+sidebar_title: Running E2E tests
+---
+
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+
```

Comment:
```suggestion

> **Warning**: EAS Build support for E2E testing is in a *very early* state. The intention of this guide is to explain how you can run E2E tests on the service today, without all of the affordances that we plan to build in the future. This guide will evolve over time as support for testing workflows in EAS Build improves.

 >  **Android not yet supported**: EAS Build does not yet support running tests on an Android Emulator — this will be coming soon.
 
```